BrowserPicker/MainWindow.xaml.cs

```diff
@@ -38,6 +38,7 @@ private void MainWindow_OnKeyUp(object sender, KeyEventArgs e)
 					case Key.D7: n = 7; break;
 					case Key.D8: n = 8; break;
 					case Key.D9: n = 9; break;
+					case Key.C: Clipboard.SetText(ViewModel.TargetURL); return;
```

Comment:
sneaky bonus feature? ^_^

Reply:
Haha - yes, it made life easier when working on the Teams links as it provided a quick way to copy the link :-)